deepface/modules/preprocessing.py

```diff
@@ -129,3 +143,58 @@ def normalize_input(img: np.ndarray, normalization: str = "base") -> np.ndarray:
         raise ValueError(f"unimplemented normalization type - {normalization}")
 
     return img
+
+
+def extract_frames_with_faces(video_bytes):
+  """Extracts frames with faces from a video as base64 bytes.
+
+  Args:
+    video_bytes: The video bytes.
+
+  Returns:
+    A list of base64-encoded frames with faces.
+  """
+  video_bytes = base64.b64decode(video_bytes)
+
+  # Create a file-like object to write the video to.
+  hash = str(random.getrandbits(128))
+  with io.open("./"+ hash + ".mp4", "wb") as f:
+    f.write(video_bytes)
+  # Decode the video bytes.
+  video_capture = cv2.VideoCapture("./"+ hash + ".mp4")
+
+  # Create a face detector.
+  face_detector = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_frontalface_default.xml')
```

Comment:
DeepFace.extract_faces should be used here instead of custom opencv implementation

Reply:
Thanks for reminding, will be update in the next version.